packages/atomic/.snyk

```diff
@@ -0,0 +1,10 @@
+# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
+# ignores vulnerabilities until expiry date; change duration by modifying expiry date
+ignore:
+  'SNYK:LIC:NPM:SALESFORCE-UX:DESIGN-SYSTEM:UNKNOWN':
+    - '*':
+        reason: None Given
```

Comment:
Please add a reason.
In your case, the licence is "LICENCE_NAME" and this is an approved licence for your use-case (you can also link to the licence text, which is always great and saves time for audit)